app/assets/javascripts/discourse/tests/acceptance/user-preferences-interface-test.js

```diff
@@ -309,6 +318,16 @@ acceptance(
         session.userDarkSchemeId.toString(),
         "resets dark scheme dropdown"
       );
+
+      await visit("/u/charlie/preferences/interface");
+
+      await selectKit(".light-color-scheme .combobox").expand();
+      await selectKit(".light-color-scheme .combobox").selectRowByValue(3);
+
+      assert.notOk(
+        document.querySelector("link#cs-preview-light"),
+        "stylesheet not loaded"
+      );
```

Comment:
Instead of adding this under the ` test("light and dark color scheme pickers"`, I think it'll be helpful if we can provide the context of what we're testing under a new `test` context. That will help to describe what we're testing. Right now, if I just look at the lines which are being added, I do not have any context about why the assertions need to exist.